src/utils/cryptography.js

```diff
@@ -62,16 +62,25 @@ class Crypto {
 	}
 
 	encryptPassphrase({ passphrase, password }) {
-		return this.liskCrypto.encryptPassphraseWithPassword(passphrase, password);
+		const encryptedPassphraseObject = this.liskCrypto.encryptPassphraseWithPassword(
+			passphrase,
+			password,
+		);
+		const encryptedPassphrase = this.liskCrypto.stringifyEncryptedPassphrase(
+			encryptedPassphraseObject,
+		);
+		return { encryptedPassphrase };
 	}
 
-	decryptPassphrase({ cipher, iv, password }) {
-		return {
-			passphrase: this.liskCrypto.decryptPassphraseWithPassword(
-				{ cipher, iv },
-				password,
-			),
-		};
+	decryptPassphrase({ encryptedPassphrase, password }) {
+		const encryptedPassphraseObject = this.liskCrypto.parseEncryptedPassphrase(
```

Comment:
I wonder why `liskCrypto` is still here...
I remember changing everything at once, but maybe i missed some.
Probably it's good time to completely say goodbye to `liskCrypto`

Reply:
https://github.com/LiskHQ/lisky/issues/516